dev/devicelab/bin/tasks/commands_test.dart

```diff
@@ -13,10 +13,10 @@ import 'package:flutter_devicelab/framework/adb.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/utils.dart';
 
-const int kObservatoryPort = 8888;
-
 void main() {
   task(() async {
+    final int vmServicePort = await findAvailablePort();
```

Comment:
Cool; I updated this to let the app bind to a free port, and discover the port from the text from the launched app.